cos/static/modal.js

```diff
@@ -35,4 +35,8 @@ window.onload = function() {
             'max-height':'100%'
         });
     });*/
-};
\ No newline at end of file
+};
+
+function dismiss_modal() {
```

Comment:
This function is empty and can be removed

Reply:
fixed